js/src/controls/SplitMapControl.js

```diff
@@ -20,30 +20,35 @@ var LeafletSplitMapControlModel = LeafletControlModel.extend({
     })
 });
 
+function asArray (arg) {
+  return  Array.isArray(arg) ? arg : [arg]
```

Comment:
Wrong indentation (we use 4 spaces in ipyleaflet). And you have an extra space between `return` and `Array`.